src/server/svr_jobfunc.c

```diff
@@ -1373,7 +1373,7 @@ svr_chkque(job *pjob, pbs_queue *pque, char *hostname, int mtype)
 
 	if (pque->qu_attr[QE_ATR_AclGroupEnabled].at_val.at_long)
 		if (acl_check(&pque->qu_attr[QE_ATR_AclGroup],
-			pjob->ji_wattr[(int)JOB_ATR_egroup].at_val.at_str,
+			pjob->ji_wattr[(int)JOB_ATR_euser].at_val.at_str,
```

Comment:
Yes that is my intention. OK, I changed the implementation of gacl_match(). It uses getpwnam() and getgrouplist() now.